src/test/java/edu/cornell/kfs/sys/databaseviews/fixture/ViewsIntegFixture.java

```diff
@@ -0,0 +1,66 @@
+package edu.cornell.kfs.sys.databaseviews.fixture;
+
+import java.util.HashMap;
+
+public enum ViewsIntegFixture {
+    COLLEGE_ORG_HRCY_V("SELECT FIN_COA_CD, ORG_CD, ORG_NM, COLLEGE_FIN_COA_CD, COLLEGE_ORG_CD, COLLEGE_ORG_NM FROM KFS.COLLEGE_ORG_HRCY_V WHERE ORG_CD = '0100'",
+            new HashMap<String, String>(){{
+                put("ORG_CD", "0100");
+                put("COLLEGE_ORG_NM", "Agriculture and Life Sciences");
+                put("COLLEGE_ORG_CD", "0100");
+                put("COLLEGE_FIN_COA_CD", "IT");
+                put("FIN_COA_CD", "IT"); 
+                put("ORG_NM", "Agriculture and Life Sciences");
+            }}),
+    
+    PERSON_DEPARTMENT_INFO_V_INCOMPLETE_DATA("SELECT NET_ID, FULL_NAME, LAST_NAME, FIRST_NAME, MIDDLE_NAME, PRIMARY_DEPARTMENT, DEPARTMENT_CHART, DEPARTMENT_ORG_CD, DEPARTMENT_ORG_NAME, COLLEGE_CHART, COLLEGE_ORG_CD, COLLEGE_ORG_NAME FROM KFS.PERSON_DEPARTMENT_INFO_V WHERE NET_ID = 'imw4'",
+            new HashMap<String, String>(){{
+                put("NET_ID" ,"imw4");
+                put("DEPARTMENT_ORG_CD", null);
+                put("COLLEGE_CHART", null);
+                put("PRIMARY_DEPARTMENT", null);
+                put("DEPARTMENT_CHART", null);
+                put("FULL_NAME", "Wicks Isabel M");
+                put("DEPARTMENT_ORG_NAME", null);
+                put("MIDDLE_NAME", "M");
+                put("LAST_NAME", "Wicks");
+                put("COLLEGE_ORG_CD", null);
+                put("COLLEGE_ORG_NAME", null);
+                put("FIRST_NAME", "Isabel");
+            }}),
+    
+    PERSON_DEPARTMENT_INFO_V_COMPLETE_DATA("SELECT NET_ID, FULL_NAME, LAST_NAME, FIRST_NAME, MIDDLE_NAME, PRIMARY_DEPARTMENT, DEPARTMENT_CHART, DEPARTMENT_ORG_CD, DEPARTMENT_ORG_NAME, COLLEGE_CHART, COLLEGE_ORG_CD, COLLEGE_ORG_NAME FROM KFS.PERSON_DEPARTMENT_INFO_V WHERE NET_ID = 'nms32'",
```

Comment:
Since Nicole has left the University, I wouldn't be surprised if some of these details get nulled out or something like that